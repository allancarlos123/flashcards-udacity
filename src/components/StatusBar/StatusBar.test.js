import React from 'react';
import { create } from 'react-test-renderer';
import StatusBar from './';

describe('StatusBar', () => {
  const tree = create(<StatusBar backgroundColor="#fff" />);
  it('renders correctly', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
});