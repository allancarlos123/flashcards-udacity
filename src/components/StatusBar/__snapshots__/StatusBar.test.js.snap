// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "height": 20,
    }
  }
/>
`;
